app/packages/core/src/plugins/SchemaIO/components/SliderView.tsx

```
@@ -3,44 +1,274 @@
+          }}
+        >
+          <MenuItem value="%">%</MenuItem>
+          <MenuItem value="flt">flt</MenuItem>
+        </Select>
+      </FormControl>
+    ),
+    [data, max, min, unit, valuePrecision]
+  );
+
+  if (labelPosition === "left") {
+    delete props?.schema?.view?.label;
```

Biome

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)